spec/features/loan_spec.rb

```diff
@@ -0,0 +1,26 @@
+require 'rails_helper'
+
+RSpec.feature 'Loans', js: true do
+  let!(:documents) { create(:document, :with_file) }
+  let!(:loan_type) { create(:loan_type, :with_documents) }
+    
+  feature 'Go to loans page' do
+    scenario 'Exist an loan' do
+      visit "/loans"
```

Comment:
@ericksonmc please use route helpers and not plan URLs, this makes it harder to maintain if a URL changes.